src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -198,20 +193,18 @@ def post(self, request, project):
             if report_instance.group:
                 report_instance.notify()
 
-        user_feedback_received.send(project=report_instance.project, group=report_instance.group, sender=self)
+        user_feedback_received.send(
+            project=report_instance.project,
+            group=report_instance.group,
+            sender=self)
 
         return self.respond(serialize(report_instance, request.user, UserReportWithGroupSerializer(
             environment_func=self._get_environment_func(
                 request, project.organization_id)
         )))
 
-    def find_event_user(self, report_data):
-        try:
-            event = Event.objects.get(
-                group_id=report_data.get('group_id'),
```

Comment:
Is this guaranteed to give you the same results? You are now filtering by project and event id.
is the event id unique within a project or within a group ?

Reply:
Yes since event ID is unique within a project 
(https://github.com/getsentry/snuba/blob/master/snuba/datasets/events.py#L183)